### PR TITLE
ENCD-5978-hic-qc

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

File: src/encoded/tests/data/inserts/hic_quality_metric.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "chimeric_ambiguous": 45,
+        "assay_term_name": "HiC",
+        "quality_metric_of": ["ENCFF355OWW"],
+        "step_run": "63b1b347-f008-4103-8d20-0e12f54d1882",
+        "uuid": "0f1951c2-f301-40e9-ad6c-1ad066e4ad71",
+        "status": "in progress",
+        "lab": "bradley-bernstein",
+        "award": "U54HG006991"
+    }
+]
```
Comment (paul-sud) [inline src/encoded/tests/data/inserts/hic_quality_metric.json]: missing newline

File: src/encoded/schemas/hic_quality_metric.json [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "pct_3_prime_bias_long_range_max": {
+            "title": "Maximum of the 3' bias long rage percentage",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
```
Comment (paul-sud) [inline src/encoded/schemas/hic_quality_metric.json]: Where did `min` and `max` come from? the two numbers for the 3' bias are for long rang and short range

File: src/encoded/schemas/hic_quality_metric.json [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+            "minimum": 0
+        },
+        "pct_single_alignment": {
+            "title": "Percentage of single alignments",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "avg_insert_size": {
+            "title": "Number of ",
```
Comment (paul-sud) [inline src/encoded/schemas/hic_quality_metric.json]: Tittle seems incomplete

File: src/encoded/schemas/hic_quality_metric.json [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+            "maximum": 100.0
+        },
+        "pct_alignable_duplicates": {
+            "title": "Percentage of alignable duplicates",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "library_complexity_estimate": {
+            "title": "Number of duplicates",
```
Comment (paul-sud) [inline src/encoded/schemas/hic_quality_metric.json]: Title is inaccurate.

File: src/encoded/schemas/hic_quality_metric.json [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+        },
+        "pct_unique_hic_contacts": {
+            "title": "Percentage of unique reads in hic contacts",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+
+        "pct_3_prime_bias_long_range_min": {
+            "title": "Minimum of the 3' bias long rage percentage",
```
Comment (paul-sud) [inline src/encoded/schemas/hic_quality_metric.json]: typo, should be `range` instead of `rage`

File: src/encoded/schemas/hic_quality_metric.json [hunk 6]
```diff
@@ -0,0 +1,349 @@
...
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "pct_3_prime_bias_long_range_max": {
+            "title": "Maximum of the 3' bias long rage percentage",
```
Comment (paul-sud) [inline src/encoded/schemas/hic_quality_metric.json]: typo, should be `range` instead of `rage`

File: src/encoded/schemas/hic_quality_metric.json [hunk 7]
```diff
@@ -0,0 +1,349 @@
...
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "lior_convergence": {
+            "title": "L-I-O-R convergence",
```
Comment (paul-sud) [inline src/encoded/schemas/hic_quality_metric.json]: Could maybe explain the acronym here. LIOR = left inner outer right

**Review by paul-sud — CHANGES_REQUESTED** [review]
Mostly looks good but there are some typos and a couple places that are unclear.

File: src/encoded/tests/data/inserts/hic_quality_metric.json (hunk 1 above)
Comment (gabdank) [reply src/encoded/tests/data/inserts/hic_quality_metric.json]: added

File: src/encoded/schemas/hic_quality_metric.json (hunk 2 above)
Comment (gabdank) [reply src/encoded/schemas/hic_quality_metric.json]: fixed

File: src/encoded/schemas/hic_quality_metric.json (hunk 3 above)
Comment (gabdank) [reply src/encoded/schemas/hic_quality_metric.json]: fixed

File: src/encoded/schemas/hic_quality_metric.json (hunk 4 above)
Comment (gabdank) [reply src/encoded/schemas/hic_quality_metric.json]: fixed

File: src/encoded/schemas/hic_quality_metric.json (hunk 5 above)
Comment (gabdank) [reply src/encoded/schemas/hic_quality_metric.json]: fixed

File: src/encoded/schemas/hic_quality_metric.json (hunk 6 above)
Comment (gabdank) [reply src/encoded/schemas/hic_quality_metric.json]: fixed

File: src/encoded/schemas/hic_quality_metric.json (hunk 7 above)
Comment (gabdank) [reply src/encoded/schemas/hic_quality_metric.json]: done

**Review by paul-sud — DISMISSED** [review]

**Review by paul-sud — APPROVED** [review]

**Pull request merged**